### PR TITLE
Fixed issue 2075

### PR DESCRIPTION
**Pull request by kzangeli**
Fixed issue #2075, about no payload in 405 errors in NGSIv2 requests.
Issues #2078 and #2083 also fixed by this PR.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -29,3 +29,4 @@
 - Fix: Trailing slashes in URI path no longer silently removed, but considered and giving adequate error responses
 - Fix: Avoid rendering at the same time id and idPattern fields in NGSIv2 subscriptions
 - Fix: Error returned on GET/DELETE requests specifying Content-Type (Issue #2128)
+- Fix: Error responses with '405 Method Not Allowed' now return payload to describe the error, for v2 requests only (Issue #2075)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Maybe #2078 and #2038 should be also mentioned?

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Ah, of course. I found about those after sending the PR. Didn't react about it :-)
Thanks!

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: #2078 and #2083 added to CNR in 9d25f63

**Comment by fgalan** [comment]
LGTM

**Pull request merged**